test/soak/cauldron/Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: cauldron
+
+TAG = 1.0
```

Comment:
Yes, sure, but do I have to do that right now when I am likely to make several rapid iterations of this code before getting to a stable version?